python/ray/dag/tests/test_accelerated_dag.py

```diff
@@ -79,6 +80,35 @@ def test_basic(ray_start_regular):
     compiled_dag.teardown()
 
 
+def test_actor_multi_methods(ray_start_regular):
```

Comment:
Might be good to add some tests that check order of execution as well, especially when two tasks on the same actor have a control dependency but not a data dependency.

Reply:
Done